androidx

please help.. i tried updating to androidx.  i can not seem to get this to work.  i am getting 

.pub-cache/hosted/pub.dartlang.org/square_in_app_payments-1.0.2/android/src/main/java/sqip/flutter/internal/GooglePayModule.java:119: error: cannot find symbol
      public void onComplete(@NonNull Task<Boolean> task) {

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,7 @@
 
 ### v1.0.3 Feb 14, 2019
 
+* Compatible to AndroidX migration
 * Some updates to improve error messages.
 * Minor code style improvements.
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 ## Changelog
 
+### v1.0.3 Feb 14, 2019
+
+* Some updates to improve error messages.
+* Minor code style improvements.
+
 ### v1.0.2 Jan 22, 2019
 
 * Fix Apple Pay issue [#18](https://github.com/square/in-app-payments-flutter-plugin/issues/18#issue-401770301).

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,8 @@
 
 ### v1.0.3 Feb 14, 2019
 
-* Compatible to AndroidX migration
+* Compatible to AndroidX migration.
+* `CardEntryDidCancelCallback` is renamed to `CardEntryCancelCallback`.
 * Some updates to improve error messages.
 * Minor code style improvements.
 

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -30,7 +30,6 @@ android {
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 27
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
     lintOptions {
         disable 'InvalidPackage'

--- a/android/src/main/java/sqip/flutter/internal/CardEntryModule.java
+++ b/android/src/main/java/sqip/flutter/internal/CardEntryModule.java
@@ -67,7 +67,7 @@ final public class CardEntryModule {
               @Override
               public void run() {
                 if (cardEntryActivityResult.isCanceled()) {
-                  channel.invokeMethod("cardEntryDidCancel", null);
+                  channel.invokeMethod("cardEntryCancel", null);
                 } else if (cardEntryActivityResult.isSuccess()) {
                   channel.invokeMethod("cardEntryComplete", null);
                 }

--- a/android/src/main/java/sqip/flutter/internal/GooglePayModule.java
+++ b/android/src/main/java/sqip/flutter/internal/GooglePayModule.java
@@ -17,7 +17,6 @@ package sqip.flutter.internal;
 
 import android.app.Activity;
 import android.content.Intent;
-import android.support.annotation.NonNull;
 import com.google.android.gms.tasks.OnCompleteListener;
 import com.google.android.gms.tasks.Task;
 import com.google.android.gms.wallet.AutoResolveHelper;
@@ -116,7 +115,7 @@ final public class GooglePayModule {
     IsReadyToPayRequest isReadyToPayRequest = GooglePay.createIsReadyToPayRequest();
     googlePayClients.isReadyToPay(isReadyToPayRequest).addOnCompleteListener(new OnCompleteListener<Boolean>() {
       @Override
-      public void onComplete(@NonNull Task<Boolean> task) {
+      public void onComplete(Task<Boolean> task) {
         result.success(task.isSuccessful());
       }
     });

--- a/doc/reference.md
+++ b/doc/reference.md
@@ -99,7 +99,7 @@ to the possible results of the request.
 Parameter       | Type                                     | Description
 :-------------- | :--------------------------------------- | :-----------
 onCardNonceRequestSuccess | [CardEntryNonceRequestSuccessCallback](#cardentrynoncerequestsuccesscallback) | Invoked when card entry is completed and the SDK has processed the payment card information.
-onCardEntryCancel | [CardEntryDidCancelCallback](#cardentrydidcancelcallback) | Invoked when card entry is canceled.
+onCardEntryCancel | [CardEntryCancelCallback](#cardentrycancelcallback) | Invoked when card entry is canceled.
 
 #### Example usage
 
@@ -607,7 +607,7 @@ import 'package:square_in_app_payments/in_app_payments.dart';
   }
 ```
 ---
-### CardEntryDidCancelCallback
+### CardEntryCancelCallback
 
 Callback invoked when card entry canceled and has been closed. 
 

--- a/example/android/app/build.gradle
+++ b/example/android/app/build.gradle
@@ -25,7 +25,7 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
 
     lintOptions {
         disable 'InvalidPackage'
@@ -38,7 +38,6 @@ android {
         targetSdkVersion 27
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
 
     buildTypes {
@@ -52,10 +51,4 @@ android {
 
 flutter {
     source '../..'
-}
-
-dependencies {
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 }

--- a/example/android/build.gradle
+++ b/example/android/build.gradle
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.1'
     }
 }
 

--- a/example/android/gradle/wrapper/gradle-wrapper.properties
+++ b/example/android/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
-#Mon Nov 19 15:13:13 PST 2018
+#Tue Feb 12 17:25:27 PST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip

--- a/example/pubspec.yaml
+++ b/example/pubspec.yaml
@@ -1,7 +1,7 @@
 name: square_in_app_payments_example
 description: Demonstrates how to use the square_in_app_payments plugin.
 
-version: 1.0.0+1
+version: 1.0.1+1
 
 environment:
   sdk: ">=2.0.0-dev.68.0 <3.0.0"

--- a/ios/Classes/FSQIPApplePay.m
+++ b/ios/Classes/FSQIPApplePay.m
@@ -33,11 +33,11 @@ API_AVAILABLE(ios(11.0))
 
 // flutter plugin debug error codes
 static NSString *const FSQIPApplePayNotInitialized = @"fl_apple_pay_not_initialized";
-static NSString *const FSQIPApplePayNotSupport = @"fl_apple_pay_not_support";
+static NSString *const FSQIPApplePayNotSupported = @"fl_apple_pay_not_supported";
 
 // flutter plugin debug messages
-static NSString *const FSQIPMessageApplePayNotInitialized = @"Please initialize apple pay before you can call other methods.";
-static NSString *const FSQIPMessageApplePayNotSupport = @"This device does not have any supported Apple Pay cards. Please check `canUseApplePay` prior to requesting a nonce.";
+static NSString *const FSQIPMessageApplePayNotInitialized = @"Apple Pay must be initialized with an Apple merchant ID.";
+static NSString *const FSQIPMessageApplePayNotSupported = @"This device does not have any supported Apple Pay cards. Please check `canUseApplePay` prior to requesting a nonce.";
 
 
 @implementation FSQIPApplePay
@@ -72,8 +72,8 @@ static NSString *const FSQIPMessageApplePayNotSupport = @"This device does not h
     }
     if (!SQIPInAppPaymentsSDK.canUseApplePay) {
         result([FlutterError errorWithCode:FlutterInAppPaymentsUsageError
-                                   message:[FSQIPErrorUtilities pluginErrorMessageFromErrorCode:FSQIPApplePayNotSupport]
-                                   details:[FSQIPErrorUtilities debugErrorObject:FSQIPApplePayNotSupport debugMessage:FSQIPMessageApplePayNotSupport]]);
+                                   message:[FSQIPErrorUtilities pluginErrorMessageFromErrorCode:FSQIPApplePayNotSupported]
+                                   details:[FSQIPErrorUtilities debugErrorObject:FSQIPApplePayNotSupported debugMessage:FSQIPMessageApplePayNotSupported]]);
         return;
     }
     PKPaymentRequest *paymentRequest =

--- a/ios/Classes/FSQIPCardEntry.m
+++ b/ios/Classes/FSQIPCardEntry.m
@@ -31,6 +31,9 @@ typedef void (^CompletionHandler)(NSError *_Nullable);
 
 @end
 
+static NSString *const FSQIPCardEntryCancelEventName = @"cardEntryCancel";
+static NSString *const FSQIPCardEntryCompleteEventName = @"cardEntryComplete";
+static NSString *const FSQIPCardEntryDidObtainCardDetailsEventName = @"cardEntryDidObtainCardDetails";
 
 @implementation FSQIPCardEntry
 
@@ -58,7 +61,7 @@ typedef void (^CompletionHandler)(NSError *_Nullable);
 - (void)cardEntryViewController:(SQIPCardEntryViewController *)cardEntryViewController didObtainCardDetails:(SQIPCardDetails *)cardDetails completionHandler:(CompletionHandler)completionHandler
 {
     self.completionHandler = completionHandler;
-    [self.channel invokeMethod:@"cardEntryDidObtainCardDetails" arguments:[cardDetails jsonDictionary]];
+    [self.channel invokeMethod:FSQIPCardEntryDidObtainCardDetailsEventName arguments:[cardDetails jsonDictionary]];
 }
 
 - (void)cardEntryViewController:(SQIPCardEntryViewController *)cardEntryViewController didCompleteWithStatus:(SQIPCardEntryCompletionStatus)status
@@ -67,18 +70,18 @@ typedef void (^CompletionHandler)(NSError *_Nullable);
     if ([rootViewController isKindOfClass:[UINavigationController class]]) {
         [rootViewController.navigationController popViewControllerAnimated:YES];
         if (status == SQIPCardEntryCompletionStatusCanceled) {
-            [self.channel invokeMethod:@"cardEntryDidCancel" arguments:nil];
+            [self.channel invokeMethod:FSQIPCardEntryCancelEventName arguments:nil];
         } else {
-            [self.channel invokeMethod:@"cardEntryComplete" arguments:nil];
+            [self.channel invokeMethod:FSQIPCardEntryCompleteEventName arguments:nil];
         }
     } else {
         if (status == SQIPCardEntryCompletionStatusCanceled) {
             [rootViewController dismissViewControllerAnimated:YES completion:^{
-                [self.channel invokeMethod:@"cardEntryDidCancel" arguments:nil];
+                [self.channel invokeMethod:FSQIPCardEntryCancelEventName arguments:nil];
             }];
         } else {
             [rootViewController dismissViewControllerAnimated:YES completion:^{
-                [self.channel invokeMethod:@"cardEntryComplete" arguments:nil];
+                [self.channel invokeMethod:FSQIPCardEntryCompleteEventName arguments:nil];
             }];
         }
     }

--- a/ios/square_in_app_payments.podspec
+++ b/ios/square_in_app_payments.podspec
@@ -4,7 +4,7 @@
 Pod::Spec.new do |s|
   s.name             = 'square_in_app_payments'
   s.version          = '1.0.0'
-  s.summary          = 'A flutter plugin for Suqare In-App Payments SDK.'
+  s.summary          = 'A flutter plugin for Square In-App Payments SDK.'
   s.description      = <<-DESC
 An open source Flutter plugin for calling Square's native In-App Payments SDK to take in-app payments.
                        DESC

--- a/ios/square_in_app_payments.podspec
+++ b/ios/square_in_app_payments.podspec
@@ -4,9 +4,9 @@
 Pod::Spec.new do |s|
   s.name             = 'square_in_app_payments'
   s.version          = '1.0.0'
-  s.summary          = 'A flutter plugin for suqare in-app payments.'
+  s.summary          = 'A flutter plugin for Suqare In-App Payments SDK.'
   s.description      = <<-DESC
-A flutter plugin for suqare in-app payments.
+An open source Flutter plugin for calling Square's native In-App Payments SDK to take in-app payments.
                        DESC
   s.homepage         = 'https://github.com/square/in-app-payments-flutter-plugin'
   s.license          = { :file => '../LICENSE' }

--- a/lib/in_app_payments.dart
+++ b/lib/in_app_payments.dart
@@ -20,7 +20,7 @@ import 'package:meta/meta.dart';
 import 'models.dart';
 import 'src/serializers.dart';
 
-typedef CardEntryDidCancelCallback = void Function();
+typedef CardEntryCancelCallback = void Function();
 typedef CardEntryCompleteCallback = void Function();
 typedef CardEntryCardNonceRequestSuccessCallback = void Function(
     CardDetails result);
@@ -45,7 +45,7 @@ class InAppPayments {
   static final _standardSerializers =
       (serializers.toBuilder()..addPlugin(StandardJsonPlugin())).build();
 
-  static CardEntryDidCancelCallback _cardEntryDidCancelCallback;
+  static CardEntryCancelCallback _cardEntryCancelCallback;
   static CardEntryCardNonceRequestSuccessCallback
       _cardEntryCardNonceRequestSuccessCallback;
   static CardEntryCompleteCallback _cardEntryCompleteCallback;
@@ -66,8 +66,8 @@ class InAppPayments {
     try {
       switch (call.method) {
         case 'cardEntryCancel':
-          if (_cardEntryDidCancelCallback != null) {
-            _cardEntryDidCancelCallback();
+          if (_cardEntryCancelCallback != null) {
+            _cardEntryCancelCallback();
           }
           break;
         case 'cardEntryDidObtainCardDetails':
@@ -141,8 +141,8 @@ class InAppPayments {
 
   static Future startCardEntryFlow(
       {CardEntryCardNonceRequestSuccessCallback onCardNonceRequestSuccess,
-      CardEntryDidCancelCallback onCardEntryCancel}) async {
-    _cardEntryDidCancelCallback = onCardEntryCancel;
+      CardEntryCancelCallback onCardEntryCancel}) async {
+    _cardEntryCancelCallback = onCardEntryCancel;
     _cardEntryCardNonceRequestSuccessCallback = onCardNonceRequestSuccess;
     await _channel.invokeMethod('startCardEntryFlow');
   }

--- a/lib/in_app_payments.dart
+++ b/lib/in_app_payments.dart
@@ -65,7 +65,7 @@ class InAppPayments {
   static Future<dynamic> _nativeCallHandler(MethodCall call) async {
     try {
       switch (call.method) {
-        case 'cardEntryDidCancel':
+        case 'cardEntryCancel':
           if (_cardEntryDidCancelCallback != null) {
             _cardEntryDidCancelCallback();
           }

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: square_in_app_payments
 description: An open source Flutter plugin for calling Square’s native In-App Payments SDK to take in-app payments on iOS and Android.
-version: 1.0.2
+version: 1.0.3
 author: Square Flutter Team <flutter-team@squareup.com>
 homepage: https://github.com/square/in-app-payments-flutter-plugin
 

